rtsf-at-checkout-device-scale/cmd/res/profiles/device-scale.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright © 2022 Intel Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+name: "cas-scale"
```

Comment:
Also rename the file to `cas-scale.yaml` for consistency and get away from naming it with the service name.